client/app/components/Input/InputError.jsx

```diff
@@ -0,0 +1,26 @@
+// @flow
```

Comment:
Let's write a unit test for this, `client/app/components/Input/__tests__/InputLabel.spec.jsx` is a good example to follow.